scripts/apps/search/directives/ItemList.js

```diff
@@ -1653,6 +1669,10 @@ export function ItemList(
                     var createItem = function createItem(itemId) {
                         var item = this.state.itemsById[itemId];
                         var task = item.task || {desk: null};
+                        var versionCreatorName;
```

Comment:
I noticed displayName as an attribute was being added by React for each class - hence named this to modifiedUserName() instead.